docs/book/src/plugins/available-plugins.md

```diff
@@ -0,0 +1,7 @@
+# Available Plugins
+
+This section details how to use the currently available plugins in Kubebuilder. 
+
+</aside>
```

Comment:
Where we are open the aside? 
why do we need it?

Reply:
Woops, I copied that over from the `plugins.md` - will remove it, good catch 👍